public-docs/installation-reaction-platform.md

```diff
@@ -0,0 +1,105 @@
+| ------------------------------------------------------------------------------------------ | -------------------------------------------------------------- |
+| [`reaction`](https://github.com/reactioncommerce/reaction): GraphQL API                    | [localhost:3000/graphpql-alpha](localhost:3000/graphql-alpha) |
+| [`reaction`](https://github.com/reactioncommerce/reaction): GraphQL API playground         | [localhost:3000/graphiql](localhost:3000/graphiql)             |
+| [`reaction`](https://github.com/reactioncommerce/reaction): Classic UI                     | [localhost:3000](localhost:3000)                               |
+| [`reaction`](https://github.com/reactioncommerce/reaction): MongoDB                           | [localhost:27017](localhost:27017)                             |
+| [`reaction-hydra`](https://github.com/reactioncommerce/reaction-hydra): oryd/hydra                     | [localhost:4444](localhost:4444)                               |
+| [`reaction-next-starterkit`](https://github.com/reactioncommerce/reaction-next-starterkit) | [localhost:4000](localhost:4000)                               |
+
+7. Congrats 🎉 Now you're running the entire suite of Reaction Platform services and ready to start developing.
+
+## Developing with Reaction Platform
+
+Once you've bootstrapped the entire Reaction development in Docker, use `make start` to start all containers.
+
+### Reaction Platform commands
+
+Run these commands from the `reaction-platform` directory:
+
+| Command                    | Description                                                                           |
+| -------------------------- | ------------------------------------------------------------------------------------- |
+| `make`                     | Boostraps the entire Reaction development environment in Docker.                      |
+| `make stop`                | Stops all containers.                                                                 |
+| `make start`               | Starts all containers.                                                                |
+| `make rm`                  | Removes all containers. Volumes are not removed.                                      |
+| `make clean`               | Removes all containers, networks, and volumes. Any volume data will be lost.          |
+| `make init-<project-name>` | Example: `make init-reaction-next-starterkit`. Does clone/setup for a single project. |
+
+Learn more about [Reaction Platform](https://github.com/reactioncommerce/reaction-platform).
+
+### Docker commands
+
+Run these commands from each respective application's directory:
+
+| Command                  | Description                         |
+| ------------------------ | ----------------------------------- |
+| `docker-compose logs -f` | Display logs and follow log output. |
```

Comment:
Looks like this is still maybe WIP? We should probably document start, stop, and destroy commands.

Reply:
We have a lotta Docker documented here already:  ```Learn more about [developing in Docker](https://docs.reactioncommerce.com/docs/next/installation-docker-development#run-the-apps) and [troubleshooting Docker development](https://docs.reactioncommerce.com/docs/next/troubleshooting-development#docker-issues). ```

This part is more like a very brief overview to get started, and the TLDR is linked below on line 101.